assets/bottom_navigation/apps.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="17" height="14" viewBox="0 0 17 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
:wrench: How about moving this to `assets/icons/apps`? This might be reused in other places. Same for other icons.

Reply:
As the icons are not used as reusable components in Figma, I don't know if that's a good idea.